changelog/dmd.truncating.conversion.dd

```diff
@@ -0,0 +1,15 @@
+Implicit integer conversions in `int op= float` assignments has been deprecated
+
+This is to prevent potential mistakes when `op=` assignment would implicitly
+truncated the right hand side expression from a non-zero value to zero.
+```
+uint a;
+float b = 0.1;
+a += b;	// Deprecation: `uint += float` is performing truncating conversion
```

Comment:
Searching for tabs
./changelog/pending.dd:165:a += b;	// Deprecation: `uint += float` is performing truncating conversion